Dockerfile

```diff
@@ -14,12 +14,11 @@ RUN pip2 install --upgrade pip && \
 
 # Install the ffig codebase. Use `cd` here to avoid several WORKDIR layers.
 RUN cd /home && \
-    curl -LOk https://github.com/FFIG/ffig/archive/master.zip && \
-    unzip master.zip && \
-    rm -f master.zip
+    git clone -b master --recurse-submodules https://github.com/FFIG/ffig.git && \
+    touch __init__.py ffig/__init__.py
```

Comment:
Does this create an `__init__.py` in `/home`?

I can see why you'd add one inside the cloned `ffig` (although maybe modifying `PYTHONPATH` would make more sense?) but I can't see any reason to have an `__init__.py` at the toplevel of `/home`.